src/plot-data.ts

```diff
@@ -2920,14 +2907,17 @@ export class Interactions {
       var min = Math.max(Math.min((mouse1X - plot_data.axis_x_start)/(plot_data.axis_x_end - plot_data.axis_x_start), (mouse2X - plot_data.axis_x_start)/(plot_data.axis_x_end - plot_data.axis_x_start)), -0.01);
       var max = Math.min(Math.max((mouse1X - plot_data.axis_x_start)/(plot_data.axis_x_end - plot_data.axis_x_start), (mouse2X - plot_data.axis_x_start)/(plot_data.axis_x_end - plot_data.axis_x_start)), 1);
     }
-    plot_data.rubber_bands[selected_axis_index].minValue = min;
-    plot_data.rubber_bands[selected_axis_index].maxValue = max;
     var realCoord_min = plot_data.axis_to_real_coords(min, plot_data.axis_list[selected_axis_index]['type_'], plot_data.axis_list[selected_axis_index]['list'], plot_data.inverted_axis_list[selected_axis_index]);
     var realCoord_max = plot_data.axis_to_real_coords(max, plot_data.axis_list[selected_axis_index]['type_'], plot_data.axis_list[selected_axis_index]['list'], plot_data.inverted_axis_list[selected_axis_index]);
     var real_min = Math.min(realCoord_min, realCoord_max);
     var real_max = Math.max(realCoord_min, realCoord_max);
-    plot_data.add_to_rubberbands_dep(new RubberBand(plot_data.axis_list[selected_axis_index]['name'], real_min, real_max));
+
+    plot_data.rubber_bands[selected_axis_index].minValue = real_min;
```

Comment:
I've seen this done twice in the code. COuld then be a function or a rubberband method

Reply:
Not sure yet. I keep it in my mind